spec/OpenLineage.json

```diff
@@ -380,7 +383,29 @@
       ],
       "type": "object"
     },
-
+    "OutputStatisticsOutputDatasetFacet": {
```

Comment:
I think if we're adding facets to json schema, we need to mention them in the docs here too:

https://github.com/OpenLineage/OpenLineage/blob/main/spec/OpenLineage.md#standard-facets

Reply:
Yes! Thank you for pointing this out

Reply:
I just added it.